src/integrations/prefect-redis/prefect_redis/messaging.py

```diff
@@ -409,3 +427,41 @@ async def break_topic():
         publishing_mock,
     ):
         yield
+
+
+async def trim_stream_to_lowest_delivered_id(stream_name: str) -> None:
```

Comment:
This also seems like it should be private-ish.
```suggestion
async def _trim_stream_to_lowest_delivered_id(stream_name: str) -> None:
```